hacks/sclang-dirt.scd

```diff
@@ -29,15 +29,20 @@ var clumps = names.asArray.scramble.curdle(0.1);
 Pdef(\x,
 	Pbind(
 		\type, \dirt,
-		\s, Pn(Plazy { var n = clumps.choose; Pfuncn({ n.choose }, 64) }),
+		\s, Pn(Plazy {
+			var n = clumps.choose;
+			postln(join(n, " ") ++ "\n");
```

Comment:
Is this postln meant to be here - looks liek a stray trace statement

Reply:
I found it good to see which instruments are playing at a given moment (btw. sorry, the chenge is unrelated).